wallet/src/de/schildbach/wallet/ui/TransactionDetailsDialogFragment.kt

```diff
@@ -44,45 +58,66 @@ class TransactionDetailsDialogFragment : DialogFragment() {
+                transaction_details_dialog_content_container.updateLayoutParams<RelativeLayout.LayoutParams> {
+                    topMargin = 30
+                }
+                transaction_close_btn.isVisible = false
+                transactionResultViewBinder.bind(it)
+            }
+
         } else {
             log.error("Transaction not found. TxId:", txId)
             dismiss()
             return
         }
-        view_on_explorer.setOnClickListener { viewOnBlockExplorer() }
+        open_explorer_card.setOnClickListener { viewOnBlockExplorer() }
         transaction_close_btn.setOnClickListener { dismissAnimation() }
+        close_btn.setOnClickListener { dismissAnimation() }
+        report_issue_card.setOnClickListener {
+            showReportIssue()
+        }
+
         showAnimation()
     }
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setStyle(STYLE_NORMAL, android.R.style.Theme_Translucent_NoTitleBar)
+    private fun showReportIssue() {
+        ReportIssueDialogBuilder.createReportIssueDialog(requireActivity(), WalletApplication.getInstance())
+            .buildAlertDialog().show()
     }
 
-    override fun onStart() {
-        super.onStart()
-        val width = ViewGroup.LayoutParams.MATCH_PARENT
-        val height = ViewGroup.LayoutParams.MATCH_PARENT
-        dialog?.window?.setLayout(width, height)
-        dialog?.window?.setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(STYLE_NO_FRAME, org.dash.wallet.common.R.style.FullScreenDialog)
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        dialog?.window?.apply {
+            setBackgroundDrawable(ColorDrawable(Color.TRANSPARENT))
+            requestFeature(Window.FEATURE_NO_TITLE)
+        }
         return inflater.inflate(R.layout.transaction_details_dialog, container, false)
     }
 
     private fun showAnimation() {
         val containerAnimation = AnimationUtils.loadAnimation(activity, R.anim.fade_in)
-        transaction_details_dialog_container.startAnimation(containerAnimation)
         val contentAnimation = AnimationUtils.loadAnimation(activity, R.anim.slide_in_bottom)
-        transaction_details_dialog_content_container.postDelayed({
+        val animSet = AnimationSet(false)
+        animSet.apply {
+            addAnimation(containerAnimation)
+            addAnimation(contentAnimation)
+        }
+
+        transaction_details_dialog_container.postDelayed({
             transaction_details_dialog_content_container.visibility = View.VISIBLE
-            transaction_details_dialog_content_container.startAnimation(contentAnimation)
+            transaction_details_dialog_content_container.startAnimation(animSet)
```

Comment:
I still see jerking in the animation. Most likely it's because `slide_in_bottom` doesn't actually start from the container being on the bottom, so it appears at `translation=0` for a moment. 
Also, `slide_in_bottom` already has fade-in animation itself. So you can fix this all like this (or use `container.animate()` extension method):
```suggestion
        val container = transaction_details_dialog_content_container
        val contentAnimation = AnimationUtils.loadAnimation(activity, R.anim.slide_in_bottom)
        container.translationY = container.measuredHeight.toFloat()
        container.visibility = View.VISIBLE
        container.startAnimation(contentAnimation)
```